notify-about-build-failure/index.js

```diff
@@ -98,47 +98,45 @@ const getCommunicationChannelsRequest = https.request(
       }
 
       const teams = parsedResult.data.orgUnits.nodes
+      let slackIdentifier = `!subteam^${fallbackSlackTeamId}`
 
-      teams.some(team => {
-        team.directRoles.some(({ member }) => {
-          const github = member.communicationChannels.find(
+      const member = teams
```

Comment:
```suggestion
      const teamMember = teams
```
To me `member` is too generic and hard to read